File: cms_web/src/Services/Entities/BillPaymongoEntity.ts
interface BillPaymongoEntity {
  bill_paymongo_pk?: string;
  id?: string;
  event_type?: string;
  source_type?: string;
  amount?: number;
  city?: string;
  country?: string;
  line1?: string;
  line2?: string;
  postal_code?: string;
  state?: string;
  email?: string;
  phone?: string;
  currency?: string;
  livemode?: boolean;
  checkout_url?: string;
  failed_url?: string;
  success_url?: string;
  status?: string;
  type?: string;
  created_at?: number;
  updated_at?: number;
  paid_at?: number;
  available_at?: number;
  consult_req_pk?: string;
  fee?: number;
  pay_descrip?: string;
  pay_net_amount?: number;
  pay_payout?: string;
  pay_src_id?: string;
  pay_src_type?: string;
  pay_statement_descrip?: string;

  //card
  payment_intent_id?: string;
  origin?: string;
  tax_amount?: string;
  refunds?: Array<any>;
  taxes?: Array<any>;
  access_url?: string;
  client_key?: string;

  card_src_id?: string;

  card_src_type?: string;
  card_src_brand?: string;
  card_src_country?: string;
  card_src_last4?: string;

  logged_at?: string | Date;
}

export interface BillPaymongoTableModel {
  limit: number;
  count: number;
  begin: number;
  table: Array<BillPaymongoEntity>;
}

export default BillPaymongoEntity;

File: cms_server/cms_server/Entities/BillPaymongoEntity.cs
using System;
using System.Collections.Generic;

namespace cms_server.Entities
{
    public class BillPaymongoEntity
    {
        public string bill_paymongo_pk { get; set; }
        public string id { get; set; }
        public string event_type { get; set; }
        public string source_type { get; set; }
        public int? amount { get; set; }
        public string city { get; set; }
        public string country { get; set; }
        public string line1 { get; set; }
        public string line2 { get; set; }
        public string postal_code { get; set; }
        public string state { get; set; }
        public string email { get; set; }
        public string phone { get; set; }
        public string currency { get; set; }
        public bool? livemode { get; set; }
        public string checkout_url { get; set; }
        public string failed_url { get; set; }
        public string success_url { get; set; }
        public string status { get; set; }
        public string type { get; set; }
        public int? created_at { get; set; }
        public int? updated_at { get; set; }
        public int? paid_at { get; set; }
        public int? available_at { get; set; }
        public string consult_req_pk { get; set; }
        public int? fee { get; set; }
        public string pay_descrip { get; set; }
        public int? pay_net_amount { get; set; }
        public string pay_payout { get; set; }
        public string pay_src_id { get; set; }
        public string pay_src_type { get; set; }
        public string pay_statement_descrip { get; set; }

        //card
        public string payment_intent_id { get; set; }
        public string origin { get; set; }
        public string tax_amount { get; set; }
        public List<dynamic> refunds { get; set; }
        public List<dynamic> taxes { get; set; }
        public string access_url { get; set; }
        public string client_key { get; set; }


        public string card_src_id { get; set; }

        public string card_src_type { get; set; }
        public string card_src_brand { get; set; }
        public string card_src_country { get; set; }
        public string card_src_last4 { get; set; }


        public DateTime? logged_at { get; set; }



    }
}
